controllers/policyset/policyPredicate.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package controllers
+
+import (
+	"k8s.io/apimachinery/pkg/api/equality"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+)
+
+// we only want to watch for policy objects with status.placement field change
```

Comment:
// we only want to watch for policy objects with status~~.placement~~ field change

Reply:
Also care about create and delete so this comment is not entirely correct